solutions/swb-app/src/audit/swbAuditExtractor.test.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Extractor } from '@aws/workbench-core-audit';
+import { AuthenticatedUser } from '@aws/workbench-core-authorization';
+import { Request, Response } from 'express';
+import SwbAuditExtractor from './swbAuditExtractor';
+
+describe('SwbAuditExtractor', () => {
+  let swbAuditExtractor: Extractor;
+  const mockMethod = 'POST';
```

Comment:
Jest recommends setting up all this data in the `beforeEach` to ensure that you can override things properly (especially in cases where there is async work happening). Not a blocker if you don't have time and everything is working right now but just something to keep in mind if you start seeing weird issues.

Reply:
Ah that's good to know. I'm going to move the constants into the test block itself since those variables are only being used in that test. Later on, if other tests need those variables, it can be moved to the `beforeEach` block.